cfgmgr/portmgr.cpp

```diff
@@ -213,11 +264,16 @@ void PortMgr::doTask(Consumer &consumer)
                 writeConfigToAppDb(alias, field_values);
             }
 
+            
+
             if (!portOk)
             {
                 SWSS_LOG_INFO("Port %s is not ready, pending...", alias.c_str());
 
-                /* Retry setting these params after the netdev is created */
+
+                writeConfigToAppDb(alias, "mtu", mtu);
```

Comment:
why do you need to explicitly call writeConfigToAppDb for these fields? It is done in the following lines.